pjsip/include/pjsua-lib/pjsua.h

```diff
@@ -594,6 +594,13 @@ typedef struct pjsua_stream_stat
     /** Jitter buffer statistic. */
     pjmedia_jb_state    jbuf;
 
+#if defined(PJMEDIA_HAS_OPUS_CODEC) && (PJMEDIA_HAS_OPUS_CODEC!=0)
+
+    /** Codec statistics. */
+    pjmedia_codec_opus_stat  opus_stat;
```

Comment:
We need to have ability to fetch opus_stat from pjsua2 too. Could you please add this structure to the `StreamStat` from `call.hpp`?

Reply:
I completely forgot that `pjsua2` takes extra effort to get the values of the base struct. I will fix it.

Reply:
Thanks!